src/bin/tikv-server.rs

```diff
@@ -71,8 +71,8 @@ fn get_string_value<F>(short: &str,
 fn initial_log(matches: &Matches, config: &toml::Value) {
     let level = get_string_value("L",
                                  "server.log-level",
-                                 &matches,
-                                 &config,
+                                 matches,
```

Comment:
If you meet compile warning later, please fire a PR to fix first, not combining it with other PRs.